### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by jsoref**
This is mostly because the last item in CONTRIBUTING is incorrect, but using markdown links makes the file more useful...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ Bun is written mostly in Zig, but WebKit & JavaScriptCore (the JavaScript engine
 Today (Feburary 2023), Bun's codebase has five distinct parts:
 
 - JavaScript, JSX, & TypeScript transpiler, module resolver, and related code
-- JavaScript runtime (`src/bun.js/`)
-- JavaScript runtime bindings (`src/bun.zig/bindings/**/*.cpp`)
-- Package manager (`src/install/`)
-- Shared utilities (`src/string_immutable.zig`)
+- JavaScript runtime ([`src/bun.js/`](src/bun.js/))
+- JavaScript runtime bindings ([`src/bun.zig/bindings/**/*.cpp`](src/bun.zig/bindings/))
```
Comment (jsoref) [inline CONTRIBUTING.md:13]: I should note that `src/bun.zig` isn't present in the repository, so I'm just going to assume that the build system generates it.

Comment (Jarred-Sumner) [reply CONTRIBUTING.md:13]: src/bun.zig is definitely in the repo

 https://github.com/oven-sh/bun/blob/4c3879814289f79d81aef30e55c4bad9145c2af8/src/bun.zig#L33

**Comment by Jarred-Sumner** [comment]
Thank you

File: CONTRIBUTING.md (hunk 1 above)
Comment (jsoref) [reply CONTRIBUTING.md:13]: That's a file, the string implied it was a directory. I was wondering if it meant bun.js

**Pull request merged**